frontend/src/views/Nodes/SelectValidatorPower/SelectValidatorPower.js

```diff
@@ -260,12 +241,13 @@ class SelectValidatorPower extends Purchase {
             <AppAlert
               className="max-alert"
               variant="primary"
-              title={<h4 className="alert-max">*More Validator Power?</h4>}
+              title={<h4 className="alert-max">About VP, Validator Power:</h4>}
             >
               <p className="alert-max">
-                If your node requires more than {formatNumbers(max)} PKT, please{" "}
-                <a href="mailto:dashboard@pokt.network">contact us</a> directly to find a solution
-                specially designed for your node.
+                Each VP unite purchased on the Pocket Dashboard has a representation as POKT token on the Pocket Network. 1VP=  1POKT
```

Comment:
Each VP unit purchased on the Pocket Dashboard is represented by a POKT token on the Pocket Network. 1 VP =  1 POKT